GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1035,15 +1040,22 @@ void OnRevisionRead(GitRevision revision)
 
                 return;
 
-                void AddArtificialRevisions(ObjectId filteredCurrentCheckout)
+                void AddArtificialRevisionsIfPossible(bool addParentGraph)
```

Comment:
```suggestion
                void AddArtificialRevisions(bool addParentGraph)
```

Reply:
The common checks are in this method so it adding also for non fatal reasons (null, empty etc) so I believe the name should be changed.